src/renderSlotComponents.tsx

```diff
@@ -96,7 +96,11 @@ const ConnectedSlot = connect(
 )(SlotRendererPure)
 
 export function SlotRenderer<T>(props: SlotRendererConnectedProps<T>): React.ReactElement<SlotRendererConnectedProps<T>> {
-    return <ConnectedSlot {...props} />
+    const [renderCount, setRenderCount] = useState(0)
+    useEffect(() => {
+        props.slot.subscribe(() => setRenderCount(count => count + 1))
+    }, [])
```

Comment:
This looks like a memory leak..

Reply:
Can you clarify why?

Reply:
Ah, maybe you also meant because of the lack of unsubscribe on `unmount`? I will add it! :)

Reply:
Added cleanup - @shirlynwix @itsh01 please re-review